crypto/bip32/child/share_manager.go

```diff
@@ -130,7 +133,7 @@ func (sHolder *shareManager) ComputeHardenedChildShare(childIndex uint32, second
 	}, nil
 }
 
-func (sHolder *shareManager) ComputeNonHardenedChildShare(childIndex uint32) (*childShare, error) {
+func (sHolder *shareManager) ComputeNonHardenedChildShare(childIndex uint32, bks map[string]*birkhoffinterpolation.BkParameter, selfId string) (*childShare, error) {
```

Comment:
- Compute cos and share in ComputeHardenedChildShare/ComputeNonHardenedChildShare
- Init bks/selfId in NewShareManager